src/mixxxapplication.cpp

```diff
@@ -207,10 +218,47 @@ bool MixxxApplication::notify(QObject* target, QEvent* event) {
 #endif // QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
 #endif // Q_OS_MAC
 
+bool MixxxApplication::notify(QObject* target, QEvent* event) {
+    // All touch events are translated in two simultaneous events one for
+    // target QWidgetWindow and one for the target QWidget
+    // A second touch becomes a mouse move without additional press and release
+    // events
```

Comment:
```suggestion
    // All touch events are translated into two simultaneous events: one for
    // the target QWidgetWindow and one for the target QWidget.
    // A second touch becomes a mouse move without additional press and release
    // events.
```
Is this just describing the behavior of Qt? Is this not described in Qt documentation?

Reply:
I have not found a written documentation about these implementation details. This knowledge was gained by debugging the Qt source.